presto-docs/src/main/sphinx/security/salesforce.rst

```diff
@@ -0,0 +1,78 @@
+============================
+Salesforce Authentication
+============================
+
+Presto can be configured to enable frontend password authentication over
+HTTPS for clients, such as the CLI, or the JDBC and ODBC drivers. The
+username and password (or password and :ref:`security token` concatenation)
+are validated by having the Presto coordinator perform a login to Salesforce.
+
+This allows Presto admins to enable users to authenticate to Presto via their Salesforce
+basic credentials. This is *not* a Salesforce connector, and does not allow users to query
+Salesforce data. It's simply a means by which users can authenticate to Presto, similar
+to :doc:`ldap` or :doc:`password-file`.
+
+Salesforce Authenticator Configuration
+--------------------------------------
+
+Enable Salesforce authentication by creating an
+``etc/password-authenticator.properties`` file on the coordinator:
+
+.. code-block:: none
+
+    password-authenticator.name=salesforce
+    salesforce.allowed-organizations=allowed salesforce org id's, or the word all
+
+The following configuration properties are available:
+
+====================================   ==============================================
```

Comment:
This could (maybe should) be formatted as a list-table or a definition list.

Reply:
E.g. see https://github.com/prestosql/presto/blob/master/presto-docs/src/main/sphinx/connector/hive-caching.rst

Reply:
@mosabua I copied the table at https://prestosql.io/docs/current/security/password-file.html#password-authenticator-configuration.  Happy to change it if you'd prefer.  Please advise.

Reply:
lets leave it for now I guess ..